packages/x-date-pickers/src/internals/hooks/useField/useField.types.ts

```diff
@@ -287,29 +291,19 @@ export interface FieldValueManager<TValue, TDate, TSection extends FieldSection,
    * @returns {string} The string value to render in the input.
    */
   getValueStrFromSections: (sections: TSection[]) => string;
-  /**
```

Comment:
I'm moving this logic inside the active date manager.
That way, all the logic related to the active date is inside one object.